splunk/dasg_metrics_dashboard.xml

```diff
@@ -381,12 +425,63 @@
       <title>4. TOTAL V2 FHIR Calls Made (w/ successful status)</title>
       <single>
         <search>
-          <query>| makeresults | eval total=$tokFhirV1OKCount$ + $tokFhirV2OKCount$ | table total</query> 
+          <query>| makeresults | eval total=$tokFhirV1OKCount$ + $tokFhirV2OKCount$ | table total</query>
         </search>
         <option name="drilldown">all</option>
         <option name="height">226</option>
         <option name="refresh.display">progressbar</option>
       </single>
     </panel>
   </row>
-</form>
+  
+  <!--SANDBOX Metrics start-->
+  
+  <row>
+    <panel>
+      <html>
+       <u1> <h3>
+            <b>SBX DASG Metrics Logging Event Counts for Group Timestamp - $tokGroupTimestamp$</b>
```

Comment:
@oragame This is displaying the group timestamp for PROD instead of Sandbox.

Wondering if you could set the "tokGroupTimestampSandbox" token in this panel or before?  Then can use it in later in 1.  (2. is already using it) panels. And will be ready for additional stats later.

Reply:
Another nice catch, thanks Dave! I'll get this corrected